src/main/resources/default/taglib/t/deleteButton.html.pasta

```diff
@@ -1,11 +1,13 @@
-<i:arg type="String" name="url"/>
-<i:arg type="Page" name="page" default=""/>
+<i:arg name="url" type="String" description="Specifies the url which should be called. Is only used when page is left empty."/>
+<i:arg name="page" type="Page" default=""/>
 <i:arg name="class" type="String" default="" description="Lists additional CSS classes to apply to the field."/>
+<i:arg name="withLabel" type="boolean" default="false" description="Specifies if the button should have an appropriate label."/>
+<i:arg name="asBtnLink" type="boolean" default="true" description="Specifies if the button should be of type link."/>
```

Comment:
is this really, really necessary? or is this just for the looks of things? I really don't like such archaic flags 😇

Reply:
I am not a huge fan either, but I was short on a better solution... 
I wanted a simple toggle to change the type of the button, by also making sure that one would not be able to do too much stuff because the template is called delete button

Reply:
This is, by the way, the same reason I decided to go with the flag for the label. Its a delete button, I dont want people to screw with the lable text yet.. :D 
But maybe it might be better to support a `labelKey`